.gitignore

```diff
@@ -1,37 +0,0 @@
-# Project exclude paths
```

Comment:
These files not to be deleted. Please add them back. Ideally, in changes I should see 2 files - readme and Main